runtime/src/chpl-privatization.c

```diff
@@ -39,32 +54,28 @@ static inline int64_t max(int64_t a, int64_t b) {
 // called with non-monotonic pid's. e.g. this may be called with pid 27, and
 // then pid 2, so it has to ensure that the privatized array has at least pid+1
 // elements. Be __very__ careful if you have to update it.
-void chpl_newPrivatizedClass(void* v, int64_t pid) {
+void chpl_newPrivatizedClass(void* v, int64_t pid) {  
```

Comment:
It's good to remove trailing spaces - if you think about it there's probably a nice way to find/replace in your editor to do it.